.github/workflows/ci.yml

```diff
@@ -68,11 +67,14 @@ jobs:
       - name: Vendor Dependencies
         run: make vendor vendor.check
 
+      - name: Lint Helm Chart
+        run: make helm.lint
+
       # This action uses its own setup-go, which always seems to use the latest
       # stable version of Go. We could run 'make lint' to ensure our desired Go
       # version, but we prefer this action because it leaves 'annotations' (i.e.
       # it comments on PRs to point out linter violations).
-      - name: Lint
+      - name: Lint Go
```

Comment:
I'm a little wary of the fact that this would mean we're no longer running the build submodule variant of `golangci-lint` anywhere during our CI runs; i.e. `make reviewable` could subtly drift from this action (due to different GoLang CI versions, etc).

Do you have an idea of how much this speeds up builds? If it's only a slight gain, perhaps the duplication is preferable.

Reply:
I haven't benchmarked it that much, but to be frank it will be a matter of seconds (maybe ~10-15 sec?), but the speed was not the main factor behind the change rather than preventing seeing false negative in the CI.

A recent example of that is #1925, `e2e-tests` is failing but because of lint issues. This, just by looking at the status of the PR, gives an impression that the change actually broke something fundamental, and might make author or reviewer waste some time trying to figure out what has happened.

But this is completely in relation of your point https://github.com/upbound/build/pull/126#discussion_r520788815 , the remediation of this was to change upstream `build/` submodule and the concerns around it.